domain/timeline/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/timeline/TimelineEvent.kt

```diff
@@ -25,7 +25,7 @@ data class TimelineEvent(
   /**
    * A map of useful contextual information about the event.
    */
-  val contextualInfo: Map<TimelineEventContext, String>?,
+  val contextualInfo: Map<TimelineEventContext, String>,
```

Comment:
Minor tidying to make this non-nullable; the code that populates this field will now set an empty map rather than null (see below)